lib/wor/paginate/rspec.rb

```diff
@@ -30,7 +30,7 @@ def paginated_content
   match do |actual_response|
     response = parse_response(actual_response)
     formatter = @custom_formatter || Wor::Paginate::Formatter
-    @formatted_keys = formatter.new(MockedAdapter.new).format.as_json.keys
+    @formatted_keys = formatter.new(MockedAdapter.new, _current_url: 'http://exaple.com/').format.as_json.keys
```

Comment:
Why rubocop don't show the message of length of the line? :thinking:

Reply:
Because is not too long? :thinking:

Reply:
Mmmm this length is actually 111 :man_shrugging: but idk